recirq/otoc/loschmidt/__init__.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2021 Google
```

Comment:
Ha, it's 2022 now!  Gotcha!  
(optional since you started it in  2021)

Reply:
oh no! now people can skirt our onerous apache2 license because the copyright is invalid!